src/bam-fetcher-worker.js

```diff
@@ -880,6 +878,14 @@ const renderSegments = (
   }
 
   if (trackOptions.showCoverage) {
+    
+    const maxCoverageSamples = 8000;
```

Comment:
Does it make sense to make this a track option?

Reply:
Hm... I don't know. It is a rather technical value. The user shouldn't see a difference vs. the exact coverage. In a 150kb window, the coverage consist of 8000 rectangles (computed at equidistant positions), vs all 150000 rectangles. Thus the resolution should be high enough. I guess it only can make a difference, when the read length is very short and the window size is large. Then, entire reads might be omitted from the calculation. But since we are restricting the zoom anyway, I don't think this will be a problem.